cgs_vmc/wavefunctions.py

```diff
@@ -1080,6 +1080,80 @@ def from_hparams(
     return cls(**full_vector_params)
 
 
+class GraphConvNetwork(Wavefunction):
+  """Implementation of wavefunction as graph neural network."""
+
+  def __init__(
+      self,
+      num_layers: int,
+      num_filters: int,
+      adj: np.ndarray,
+      nonlinearity: tf.Tensor = tf.nn.relu,
+      output_activation: tf.Tensor = tf.exp,
+      name: str = 'graph_neural_network',
```

Comment:
nit: As suggested by Sonnet library, default names of snt.AbstractModule should be  equal to class name converted from CamelCase to underscore case.  i.e. in this case it would be `graph_conv_network `.